docker/compose_files/docker-compose.yml

```diff
@@ -27,11 +28,20 @@ x-codalab-env: &codalab-env
   - CODALAB_SERVER=${CODALAB_SERVER}
   - CODALAB_SHARED_FILE_SYSTEM=${CODALAB_SHARED_FILE_SYSTEM}
   - CODALAB_WORKER_MANAGER_TYPE=${CODALAB_WORKER_MANAGER_TYPE}
-  - CODALAB_WORKER_MANAGER_MAX_CPU_WORKERS=${CODALAB_WORKER_MANAGER_MAX_CPU_WORKERS}
-  - CODALAB_WORKER_MANAGER_MAX_GPU_WORKERS=${CODALAB_WORKER_MANAGER_MAX_GPU_WORKERS}
+  - CODALAB_WORKER_MANAGER_IDLE_SECONDS=${CODALAB_WORKER_MANAGER_IDLE_SECONDS}
+  - CODALAB_WORKER_MANAGER_SECONDS_BETWEEN_WORKERS=${CODALAB_WORKER_MANAGER_SECONDS_BETWEEN_WORKERS}
+  - CODALAB_WORKER_MANAGER_AWS_REGION=${CODALAB_WORKER_MANAGER_AWS_REGION}
+  - CODALAB_WORKER_MANAGER_AWS_BATCH_JOB_DEFINITION_NAME=${CODALAB_WORKER_MANAGER_AWS_BATCH_JOB_DEFINITION_NAME}
+  - CODALAB_WORKER_MANAGER_AWS_DEFAULT_CPUS=${CODALAB_WORKER_MANAGER_AWS_DEFAULT_CPUS}
+  - CODALAB_WORKER_MANAGER_AWS_DEFAULT_MEMORY_MB=${CODALAB_WORKER_MANAGER_AWS_DEFAULT_MEMORY_MB}
   - CODALAB_WORKER_MANAGER_CPU_QUEUE=${CODALAB_WORKER_MANAGER_CPU_QUEUE}
   - CODALAB_WORKER_MANAGER_GPU_QUEUE=${CODALAB_WORKER_MANAGER_GPU_QUEUE}
-  - HOSTNAME=${HOSTNAME}
```

Comment:
Why is this deleted?

Reply:
Just moved to the top of the list as the only non-codalab env var.